main.tf

```diff
@@ -1 +1,62 @@
+resource "azuread_group" "aaddc_admins" {
+  display_name     = "AAD DC Administrators"
+  security_enabled = true
+}
 
+resource "azuread_user" "aaddc_admin" {
+  user_principal_name = "fct_eds_admin@${var.domain}"
```

Comment:
I think our default is "-" not "_". Not sure if it would be fct-eds-admin or fct-eds_admin. Need to check the naming convention myself

Reply:
I think its more like that: fct_eds-admin. @QBY-ThimoLimpert what do you say?

Reply:
`fct_eds-admin` is correct. Talked to colleague, let's use `fct_black_eds` So `fct` as prefix, `black` as resource (as always) and `eds` as suffix.